prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -90,6 +93,12 @@ class FHIRConverter(
                 // conduct FHIR Transform
                 transformer?.transform(bundle)
 
+                bundle.entry.filter { it.resource is Observation }.forEach {
```

Comment:
Nit: I think think this can be
```suggestion
                bundle.entry.filterType<Observation>.forEach {
```

and then you can drop the cast

Reply:
Excellent suggestion, will do! I was really struggling to find that method call lol